qa/common/common.sh

```diff
@@ -149,7 +167,39 @@ EOF
 #
 
 function proposal_populate_dmcrypt {
-    salt-run proposal.populate encryption='dmcrypt' name='dmcrypt'
+  salt-run proposal.populate encryption='dmcrypt' name='dmcrypt'
+}
+
+#
+# functions for restarting all configured services
+#
+
+function restart_services {
+  salt-run state.orch ceph.restart
+}
+
+function mon_restarted {
+  local exit_status 
+  local expectation=$1
+  salt --static --out json -C 'I@roles:mon' cmd.shell "journalctl -u ceph-mon@* " | grep -i terminated
```

Comment:
If we're telling Salt to generate JSON output, would it make more sense to use `jq` instead of `grep` to parse the output?

Reply:
In this case, if we're just checking for the presence of a string "terminated", we could just drop `--static --out json`?

Reply:
What if there are multiple nodes with the `mon` role, and only one restarts? The test will pass?

Reply:
It will pass, yes. Iterating over hosts will definitely improve the test. Let me add it.